index.html

```diff
@@ -97,7 +117,8 @@ <h2>Tonic</h2>
 
     <section class="card-works1 works img-right">
       <div class="image-2">
-        <img  src="images/Snap2.png" alt="Card1" />
```

Comment:
- [ ] Your popup has good content, beautiful pictures, and good colors. But if possible you can improve the design by fixing the size of your images and the buttons, according to the [templet](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=34%3A1143) :+1: